test/integration/auth.spec.ts

```diff
@@ -1247,6 +1247,13 @@ describe('admin.auth', () => {
       multiFactorConfig: {
         state: 'ENABLED',
         factorIds: ['phone'],
+        // TODO: Once createTenant() is implemented on the backend, populate providerConfigs
```

Comment:
Shouldn't we add test cases to the project config update flow also , like line 1189?